README.md

```diff
@@ -42,3 +55,55 @@ To run the stack in production, you can run the following
 ```
 
 This will start the stack in production mode.
+
+## Docker Commands
```

Comment:
I think this section is not warranted. This is general docker knowledge that would be better deferred to official docker docs. Moreover, the `run_stack` script aims to abstract a lot of this away so the developer doesn't have to worry about it. If someone were to need these lower level commands, it's fair and necessary to assume some background knowledge by the user.